frontend/src/components/Forms/reports/AddReportForm.js

```diff
@@ -90,6 +92,25 @@ export class AddReportForm extends Component {
     }));
   };
 
+  updateDueDateWithDefaultDueDate = (selectedMineReportDefinition) => {
+    let formMeta = this.props.formMeta;
+    if (
+      !(
+        formMeta &&
+        formMeta.fields &&
+        formMeta.fields.due_date &&
+        formMeta.fields.due_date.touched == true
+      )
+    ) {
+      this.props.change(
+        "due_date",
+        this.props.mineReportDefinitionOptions.find(
+          (x) => x.mine_report_definition_guid == selectedMineReportDefinition
```

Comment:
Please use strict equality (`===`). This should be causing a linting error. Are you not seeing errors?

Reply:
i'm getting this `Failed to load plugin 'react' declared in 'frontend-public\.eslintrc.js': Cannot find module 'eslint-plugin-react'`. and the pre-hook is saying on .js files found.

Reply:
Oh. The same regression that @mmumby is having. I didn't realize that it was wide-spread. It's a result of the dependency upgrade. I'll push the priority up on this fix. Thanks for replying with the details